kpatch-build/create-kpatch-module.c

```diff
@@ -50,13 +50,13 @@ static void create_dynamic_rela_sections(struct kpatch_elf *kelf, struct section
 
```

Comment:
Most of these casts can be avoided by changing the variable type to `unsigned int`.